modules/plot/src/plot.cpp

```diff
@@ -291,12 +291,12 @@ namespace cv
                 //setting the min and max values for each axis
                 plotMinX = MinX;
                 plotMaxX = MaxX;
-                plotMinY = MinY;
-                plotMaxY = MaxY;
+                plotMinY = MaxY;
+                plotMaxY = MinY;
```

Comment:
We should pay attention to constructs like this. We are assigning min and maximum values to plot max and min values ... why is this needed? Could you explain in more detail. If this is the case, then the functionality should be rewritten in order to get better readable code.